ansible/playbook/tasks/roles/sqlite-to-postgres/tasks/main.yml

```diff
@@ -0,0 +1,69 @@
+---
+- name: Create Grafana backup dir
+  file:
+    path: "{{ grafana_backup_dir }}"
+    state: directory
+    owner: grafana
+    group: grafana
+    mode: '0700'
+
+- name: Stop grafana before upgrade
+  supervisorctl:
+    name: 'grafana'
+    state: stopped
+
+- name: Create grafana user
+  postgresql_user:
+    db: grafana
+    name: grafana
+    password: grafana
+    priv: 'ALL'
+    expires: infinity
+    state: present
+  when: not ansible_check_mode
+
+- name: Create actual grafana config
+  include_role:
```

Comment:
How will this work once we migrate this role to Go (pmm-update)?

Reply:
We run this role from Ansible from pmm-update. It's the same Ansible.